bdc_motor/CMakeLists.txt

```
@@ -4,6 +4,12 @@ if(CONFIG_SOC_MCPWM_SUPPORTED)
     list(APPEND srcs "src/bdc_motor_mcpwm_impl.c")
 endif()
 
+if("${IDF_VERSION_MAJOR}.${IDF_VERSION_MINOR}" VERSION_GREATER_EQUAL "5.3")
+set(priv_requires "esp_driver_pcnt" "esp_driver_mcpwm")
```

pcnt is not used by this library, we can remove it.


```suggestion
    set(priv_requires "esp_driver_mcpwm")
```